test/integration/forms/action.js

```diff
@@ -1030,21 +1030,58 @@ context('Patient Action Form', function() {
       .should('not.contain', 'patient-action/1/form/11111');
   });
 
-  specify('read only form', function() {
-    localStorage.setItem('form-subm-11111-1-22222-1', JSON.stringify({
-      updated: testTs(),
-      submission: {
-        patient: { fields: { foo: 'foo' } },
-      },
-    }));
+  specify('action done form', function() {
+    cy
+      .routeAction(fx => {
+        fx.data.id = '1';
+        fx.data.relationships.form.data = { id: '11111' };
+        fx.data.relationships.state.data = { id: '55555' };
+        fx.data.relationships['form-responses'].data = [];
+        return fx;
+      })
+      .routePatient()
+      .routeFormByAction()
+      .routeFormDefinition()
+      .routeActionActivity()
```

Comment:
Do we need `routePatient()` or `routeActionActivity()` here?

Reply:
ah right this should be a `routesFor` this branch is actually really old.. will update